hugo/content/guides/scoping/_index.md

```diff
@@ -3,24 +3,29 @@ title: "Scoping"
 weight: 100
 ---
 
-The process of scoping determines which elements can be referenced at a certain position of a document. You likely know scopes from programming, where some variables are only available from certain scopes:
+You like know scopes from programming, where some variables are only available from certain areas (such as blocks) in your program. For example, take the short Typescript snippet below. Based on the block (scope) where a variable is declared, it may or may not be available at another location in the same program.
```

Comment:
Typo on 'like'
```suggestion
You likely know scopes from programming, where some variables are only available from certain areas (such as blocks) in your program. For example, take the short Typescript snippet below. Based on the block (scope) where a variable is declared, it may or may not be available at another location in the same program.
```